src/App.js

```diff
@@ -0,0 +1,87 @@
+// import from react
+import { Component } from 'react'
+import { 
+  BrowserRouter as Router, 
+  Route, 
+  Switch, 
+  Link 
+} from "react-router-dom";
+// import from js
+import EventCard from "./components/eventCard";
+import CreateEvent from './components/createEvent'
+import EventDetails from './components/EventDetails'
+// import from css
+import '../src/css files/App.css';
+import '../src/css files/cards.css';
```

Comment:
Ideally, we want to avoid loading more than one CSS file per .js file if possible.  In addition to possible conflicts due to the cascading nature of CSS, it's also usually a sign that we could probably consolidate some CSS somewhere.  After all, since all of the components are rendered within the App.js file itself, their own CSS files (i.e. any imported directly into them) will have been applied to them when they in turn are imported by the App.

If it's the case that we had to import this here to avoid some issue with one style over-writing another when the App is live, there is probably another way to get around it without importing two style files.